src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,227 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Maestro.Data;
+using Microsoft.DotNet.ServiceFabric.ServiceHost;
+using Microsoft.Extensions.Logging;
+using Microsoft.ServiceFabric.Data;
+using Microsoft.ServiceFabric.Data.Collections;
+using Microsoft.Dotnet.GitHub.Authentication;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.DotNet.Git.IssueManager;
+using System.Text;
+using Newtonsoft.Json.Linq;
+using System.Net.Http;
+using Microsoft.TeamFoundation.Common;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace DependencyUpdateErrorProcessor
+{
+    /// <summary>
+    /// An instance of this class is created for each service replica by the Service Fabric runtime.
+    /// </summary>
+    internal sealed class DependencyUpdateErrorProcessor : IServiceImplementation
+    {
+        public DependencyUpdateErrorProcessor(
+            IReliableStateManager stateManager,
+            ILogger<DependencyUpdateErrorProcessor> logger,
+            BuildAssetRegistryContext context,
+            IOptions<DependencyUpdateErrorProcessorOptions> dependencyUpdateErrorProcessorOptions
+            )
+        {
+            StateManager = stateManager;
+            Logger = logger;
+            Context = context;
+            DependencyUpdateErrorProcessorOptions = dependencyUpdateErrorProcessorOptions.Value;
+        }
+
+        public IReliableStateManager StateManager { get; }
```

Comment:
This does match what we have in the other Maestro services. @alexperovich do you know if this is something we need to do to appease the Service fabric gods, or is it just an oversight that has spread as we use the existing services as templates when we add new ones?

If we can make them private, this could be the first service to do the right thing, and we can then think of changing the existing ones.

Reply:
Looks like there is no good reason for these to be public properties, making them private fields in this new service seems like the right thing to do,  I also opened https://github.com/dotnet/core-eng/issues/9037 so we consider changing these up for existing services